projects/desktop/src/helpers/auth.ts

```diff
@@ -1,17 +1,13 @@
 import jwtDecode from 'jwt-decode'
 
-/**
- * Function who checks if the user has a token set or not
- * @returns boolean
- */
-export const isAuthenticated = (): boolean => !!localStorage.getItem('token')
+import { getSession } from '../redux/helpers/session'
 
 /**
  * Function who retrieves the data from the token set
  * @returns object | undefined
  */
 export const getTokenData = (): object | undefined => {
-    const token: string | null = localStorage.getItem('token')
+    const token: string | null = getSession().token

```

Comment:
Instead of `string | null` can we do something like `getSession().token ?? something`?
To remove the `null` in the type of the token

Reply:
We should keep the null because if you don't have a token, having the null value makes sense